### PR TITLE
Fix media type error

### PR DESCRIPTION
**Pull request by duggalsu**

### CODE REVIEW

File: sharechat_helper.py [hunk 1]
```diff
@@ -221,7 +221,20 @@ def get_post_data(payload_dict, tag_name, tag_translation, tag_genre, bucket_nam
     verified = []
     
     for i in payload_dict.get("payload").get("d"):
-        if i.get("t") == "image":
+        if "repostId" in i:
+            print(i)
+            i["t"] = "repost"
+            if "x" in i.keys():  # if post metadata contains the text
+                get_common_metadata(i, timestamp, language, media_type, post_permalink, caption, external_shares, likes,
```
Comment (kruttikanadig) [inline sharechat_helper.py:227]: Must get_common_metadata() be conditioned on "x" being in the dict keys? We'll want to save the common metadata of reposts even if they don't have text in "x", as long as they have a repostData field with the shared post link in it

File: sharechat_helper.py [hunk 2]
```diff
@@ -221,7 +221,20 @@ def get_post_data(payload_dict, tag_name, tag_translation, tag_genre, bucket_nam
...
+                                    comments, reposts, views, profile_page, verified)
+                if "repostData" in i:
```
Comment (kruttikanadig) [inline sharechat_helper.py:229]: 2. Also unsure why scraping the repostData field is conditioned on "x" - could you clarify

Comment (kruttikanadig) [inline sharechat_helper.py:229]: 2. Also unsure why scraping the repostData field is conditioned on "x" - could you clarify please?

File: sharechat_helper.py [hunk 3]
```diff
@@ -221,7 +221,20 @@ def get_post_data(payload_dict, tag_name, tag_translation, tag_genre, bucket_nam
...
+                    shared_post_url = "https://sharechat.com/post/" + i.get("repostData").get("ph")
+                    text.append(i.get("x") + "<Repost:" + shared_post_url + ">")
```
Comment (kruttikanadig) [inline sharechat_helper.py:231]: Suggest appending the text and the shared post url as key:value pairs to enable easier database filtering and avoid having to use regex to find the links later. i.e. append {"text": <text in x>, "shared_post": <shared post url>}. Unless you forsee any issues with this

File: sharechat_helper.py (hunk 1 above)
Comment (duggalsu) [reply sharechat_helper.py:227]: I wasn't sure about this. I will make the changes.

File: sharechat_helper.py (hunk 2 above)
Comment (duggalsu) [reply sharechat_helper.py:229]: I will make the changes as above. That should resolves this too.

File: sharechat_helper.py (hunk 3 above)
Comment (duggalsu) [reply sharechat_helper.py:231]: Will implement this. However, I am testing in local mode. Can I still somehow test db integration in local mode?

Comment (kruttikanadig) [reply sharechat_helper.py:231]: Unfortunately not. I think it should work though. For local testing you could just check that the text column in [sharechat_df](https://github.com/tattle-made/sharechat-scraper/blob/master/sharechat_scrapers.py) contains values in the format {"text": <text in x>, "shared_post": <shared post url>} wherever the media type is repost. Pymongo should be able to upload them correctly to the db.

File: sharechat_helper.py (hunk 1 above)
Comment (kruttikanadig) [reply sharechat_helper.py:227]: Thanks!

**Review by kruttikanadig — APPROVED** [review]

**Pull request merged**